tests/embed/test_n2v.py

```diff
@@ -1,25 +1,223 @@
 # Copyright (c) Microsoft Corporation and contributors.
```

Comment:
Combine 2 n2v test files. Correct testing file structure should follow module structure. Embedding tests belong in tests -> embed and not in the root.